forwarder_test.go

```diff
@@ -222,11 +205,17 @@ func TestUpdateResourceIpAddressAndCertificateInfo(t *testing.T) {
 			assert.NoError(err)
 
 			err = updateResourceIpAddressAndCertificateInfo(testReq, client, resourceURL)
```

Comment:
Add assertions to validate request body sent while calling update resource by mac address.